README.md

```diff
@@ -1,70 +1,83 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Covid-19 Tracking Project
 
-## Available Scripts
+> "Covid-19 Tracking Project" is a mobile web application that displays Covid-19 tracking data from [Narrativa API](https://covid19tracking.narrativa.com/index_en.html). You can check detailed data for each country separately. It was build using React and Redux libraries.
 
-In the project directory, you can run:
 
-### `yarn start`
+![screenshot](./public/screenshot.png)
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Built With
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+- React
+- Redux
 
-### `yarn test`
+## Live demo link
```

Comment:
- [ ] Please record an additional video highlighting a piece of code on the project that you found interesting.